x-pack/plugins/secops/public/components/flyout/header/index.tsx

```diff
@@ -104,20 +106,35 @@ const statefulFlyoutHeader = pure<Props>(
 
 const emptyHistory: History[] = []; // stable reference
 
-const mapStateToProps = (state: State, { timelineId }: OwnProps) => {
-  const timelineById = defaultTo({}, timelineSelectors.timelineByIdSelector(state));
+const makeMapStateToProps = () => {
+  const getTimeline = timelineSelectors.getTimelineByIdSelector();
+  const getNotesByIds = appSelectors.notesByIdsSelector();
+  const mapStateToProps = (state: State, { timelineId }: OwnProps) => {
+    const timeline: timelineModel.TimelineModel = getTimeline(state, timelineId);
 
-  const description = getOr('', `${timelineId}.description`, timelineById);
-  const history = emptyHistory; // TODO: get notes from store via selector
-  const isFavorite = getOr(false, `${timelineId}.isFavorite`, timelineById);
-  const isLive = getOr(false, `${timelineId}.isLive`, timelineById);
-  const title = getOr('', `${timelineId}.title`, timelineById);
-  const noteIds = getOr([], `${timelineId}.noteIds`, timelineById);
-  const notesById = appSelectors.notesByIdSelector(state);
-  const notes = getApplicableNotes({ noteIds, notesById });
-  const width = getOr(defaultWidth, `${timelineId}.width`, timelineById);
+    const {
+      description = '',
+      isFavorite = false,
+      isLive = false,
+      title = '',
+      noteIds = [],
+      width = defaultWidth,
+    } = timeline;
 
-  return { description, history, isFavorite, isLive, notes, title, width };
+    const history = emptyHistory; // TODO: get notes from store via selector
```

Comment:
> // TODO: get notes from store via selector

I appreciate the TODO.  Perhaps a slight tweak to the comment, `s/notes/history`?

Reply:
it was not mine lol